Update/ura_003.txt

```diff
@@ -1913,7 +1913,7 @@ void main()
 //rあぁ、やっぱりこいつはこういう場面でダメダメだねぇ。一言『いい』って伝えるだけで十分好感度アップなのに。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "あぁ、やっぱりこいつはこういう場面でダメダメだねぇ。一言『いい』って伝えるだけで十分好感度アップなのに。",
-			NULL, "Right, that one is completely useless in situations like this, isn't he? Even though it would be more than enough to just say \"it's nice.\"", Line_Normal);
+			NULL, "Oh boy, that one  really is completely useless in situations like this, isn't he? Even though it would be more than enough to just say \"it's nice.\"", Line_Normal);
```

Comment:
```suggestion
			NULL, "Oh boy, that one really is completely useless in situations like this, isn't he? Even though it would be more than enough to just say \"it's nice.\"", Line_Normal);
```